starter-code/src/functions-and-arrays.js

```diff
@@ -1,16 +1,44 @@
 // Iteration #1: Find the maximum
 
+function maxOfTwoNumbers (a, b) {
+  return Math.max(a,b)
+}
+console.log(maxOfTwoNumbers(10,20)) 
+
 // Iteration #2: Find longest word
-const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot']; 
+
+ let lgth = 0;
+ let longest;
+
+for(let i=0; i < words.length; i++){
+    if(words[i].length > lgth){
+        lgth = words[i].length;
+        longest = words[i];
+    }
+}
+  console.log(longest)
 
 // Iteration #3: Calculate the sum
 
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
+var sum = numbers.reduce(function(a, b) { return a + b; }, 0);
+
+console.log(sum)
+
 // Iteration #4: Calculate the average
 // Level 1: Array of numbers
 const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
+function avrg(arr) {
+  const sum = arr.reduce((a, b) => a + b, 0);
```

Comment:
as i said before please dont use reduce for now, first we need to learn basic js then we gonna cover everything.
it took so much of your time and you are half done